.github/workflows/bindings.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Bindings
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  rust:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        id: toolchain
+        with:
+          toolchain: stable
+          override: true
+
+      - uses: camshaft/rust-cache@v1
+
+      - name: Generate
+        run: ./bindings/rust/generate.sh
```

Comment:
On a clean ubuntu20 docker images, had todo:
```
apt install libssl-dev pkg-config
OPENSSL_LIB_DIR=/usr/lib/x86_64-linux-gnu ./bindings/rust/generate.sh
```
to get it built- not sure if the GHA images just have more installed?

Reply:
yeah they come pretty loaded

Reply:
but I can add those as required packages as well